mithril-aggregator/src/http_server/routes/certificate_routes.rs

```diff
@@ -121,16 +121,109 @@ mod handlers {
 #[cfg(test)]
 mod tests {
     use anyhow::anyhow;
-    use mithril_common::test_utils::{apispec::APISpec, fake_data};
+    use async_trait::async_trait;
+    use mithril_common::{
+        entities::CertificatePending,
+        test_utils::{apispec::APISpec, fake_data},
+    };
+    use mithril_persistence::store::adapter::AdapterError;
+    use mithril_persistence::store::adapter::StoreAdapter;
     use serde_json::Value::Null;
-    use warp::{http::Method, test::request};
+    use warp::{
+        http::{Method, StatusCode},
+        test::request,
+    };
 
     use crate::{
-        http_server::SERVER_BASE_PATH, initialize_dependencies, services::MockCertifierService,
+        http_server::SERVER_BASE_PATH, initialize_dependencies, services::MockMessageService,
+        CertificatePendingStore,
     };
 
     use super::*;
 
+    ///////////////
+
+    pub struct MockStoreAdapter {}
+
+    #[async_trait]
+    impl StoreAdapter for MockStoreAdapter {
```

Comment:
You could probably have used the [DumbStoreAdapter](https://github.com/input-output-hk/mithril/blob/6f430364f9c76d9b98b42855d2cf3250d364a8e6/internal/mithril-persistence/src/store/adapter/dumb_adapter.rs#L27) for that?